net/babeld/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2007-2009 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=babeld
+PKG_VERSION:=1.8.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://www.irif.fr/~jch/software/files/
+PKG_HASH:=98070dc418c190f047b8d69eb47987df30ded8f0fca353c49427d3137ad08b87
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=
```

Comment:
Should be ```LICENCE```?
https://github.com/jech/babeld/blob/master/LICENCE

Reply:
Updated.